decksite/data/query.py

```diff
@@ -237,3 +237,6 @@ def card_search_where(q: str) -> str:
 
 def tournament_only_clause() -> str:
     return "ct.name = 'Gatherling'"
+
+def deck_scraper(ts: int) -> str:
+    return f') AND d.finish AND (q.changed_date > {ts} OR d.updated_date > {ts}'
```

Comment:
Requiring a value in `d.finish` will only get you tournament decks. I assume you don't want that.

Reply:
This should be a complete piece of logic not a partial with a hanging paren. The hanging paren requires the caller to have altogether too much knowledge of what this func does.

```
return 'd.finish AND (q.changed_date > {ts} OR d.updated_date > {tr})'
```

Then in the caller:

```
where = query.decks_where(…) + ' AND ' + query.decks_since(…)
```

Reply:
That's what it initially was, but there are a bunch of ORs at the end of the decks_where request, so it just ignored the OR. The tournament bit is interesting, and explains the 59000 deck issue, thank you.

Reply:
You can use parens to work around this:

```
where = '(' + query.decks_where(…) + ') AND (' + query.decks_since(…) + ')'
```